### DISCUSSION TITLE
lesion and structure segmentation

### DISCUSSION THREAD

**Discussion by qwang0225** [Project-MONAI/tutorials#1641] [Q&A]
I have a MRI brain dataset including DWI, ADC and FLAIR sequences with stroke lesions. I'm able to segment lesion from DWI/ADC. My next step is to determine if the lesion involves certain brain structure called insula. My plan was to segment insula separately and then compute if my insula masks and lesions masks overlap on the same DWI images. 

The challenge I'm facing is I'm not sure how to conduct insula segmentation. Because DWI has poor image resolution, I would have to manually segment insula on FLAIR then register onto DWI to get insula mask on DWI. Is that the right approach? Is there easier way to do this? I know monai zoo has a whole brain segmentation model, but it's trained on T1. 

Thanks

**Comment by diazandr3s** [comment]
Hi @qwang0225,

You could consider using [SynthSeg](https://github.com/BBillot/SynthSeg) to create "ground-truth" anatomy labels on MR images and then train an AI model to replicate what SynthSeg does on the modality you wish.

Hope this helps,

**Reply by qwang0225** [reply]
Thanks. It's a robust pretrained model. However, correct me if I'm wrong, SynthSeg does not seem to have insula labels.

**Reply by diazandr3s** [reply]
Are you sure? I've seen segmented brain MR images using SynthSeg with `'ctx-lh-insula' ` and `'ctx-rh-insula'` labels

**Reply by qwang0225** [reply]
I followed the [link](https://github.com/BBillot/SynthSeg/blob/master/data/labels%20table.txt ) and opened the label files.

**Reply by qwang0225** [reply]
I'll just run the model myself to see if they have insula labels

**Reply by qwang0225** [reply]
Actually they do have insula labels. If I would like to segment insula into anterior vs posterior part. Any suggestions on how to proceed?
Thanks!

**Reply by diazandr3s** [reply]
Hi @qwang0225,

Do you mean a 2-label model? I'd suggest you prepare or merge the labels to train a whole-brain segmentation model. It could work better than a 2-label model.

**Reply by qwang0225** [reply]
Thanks! 
I'm trying to do cortical parcellation, but under their models directory, there is only one file called "synthseg_1.0.h5".
After I ran, I got the error "D:\project\SynthSeg\models\synthseg_parc_2.0.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)". Any suggestions?
